frontend/dashboard/pages/Dashboard/Dashboard.module.css

```
@@ -9,6 +9,11 @@
   margin-bottom: var(--fds-spacing-10);
 }
 
+.searchContainer {
```

I suggest changing the CSS class names to `search`, since they are applied to the components themselves, not their containers.
```suggestion
.search {
```